src/lib.rs

```diff
@@ -113,3 +114,9 @@ pub use com_macros::class;
 // whether they are used by some other crate or internally
 #[doc(hidden)]
 extern crate self as com;
+
+extern crate alloc;
+
+// We re-export `alloc::boxed::Box` so that we can use `com::Box` in generated code.
+#[doc(hidden)]
+pub use alloc::boxed::Box;
```

Comment:
Perhaps it might make more sense to re-export `alloc` instead, so that we don't have to update this if the generated code needs more from the alloc crate.

Reply:
Agreed. I'll use that.